setup.cfg

```diff
@@ -53,6 +53,7 @@ addopts =
 filterwarnings =
     error
     ignore:module 'ssl' has no attribute 'OP_NO_COMPRESSION'. The Python interpreter is compiled against OpenSSL < 1.0.0. Ref. https.//docs.python.org/3/library/ssl.html#ssl.OP_NO_COMPRESSION:UserWarning
+    ignore:The loop argument is deprecated since Python 3.8, and scheduled for removal in Python 3.10.:DeprecationWarning
```

Comment:
Note for the future: This will probably be fixed in #6004. We shouldn't include it in the final patch.

Reply:
I'll remove this when I undraft it.

Reply:
I can rebase my PR after https://github.com/aio-libs/aiohttp/pull/6004/files merges.

Reply:
I rebased this and removed this change from my PR.